unlock-app/src/middlewares/walletMiddleware.js

```diff
@@ -104,24 +105,32 @@ export default function walletMiddleware({ getState, dispatch }) {
         walletService.connect(action.provider)
       } else if (action.type === CREATE_LOCK) {
         // Create the lock
-        walletService.createLock(action.lock, getState().account.address)
-        // And sign its name
-        if (config.services.storage) {
-          generateJWTToken(walletService, getState().account.address, {
-            lock: action.lock,
+
+        if (!action.lock.address) {
```

Comment:
Maybe put a comment above this to explain what is going on?

Reply:
for future-akeem or future-anyone ;)

Reply:
This feedback has now been addressed